FinHubSA/information-access
# feat: results are in pages

# Pull Request

## Scope
https://dev.azure.com/JHNMAT015/ECO5037S%20Front%20end/_backlogs/backlog/ECO5037S%20Front%20end%20Team/Backlog%20items/?workitem=145
Divided results into pages that can be moved through. 

## Work Done
1. Added in page number and arrows at bottom of results page. 
2. Changed routing to make switching between pages functional. 
3. Selected subset of results to show on each page. 

## Steps to Test
``` npm run start```
Search for something that has more than 2 results. 
Click on right page arrow.
You should see a new page. 

## Screenshot
![image](https://user-images.githubusercontent.com/80746660/136746000-52c3de89-be3c-4698-8e46-e0eddc6b57bf.png)

![image](https://user-images.githubusercontent.com/80746660/136746049-cd64fba8-8ad5-443a-89e3-da3acc09e2ab.png)
